<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow2</class>
 <widget class="QWidget" name="MainWindow2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BabySteps</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>783</width>
        <height>566</height>
       </rect>
      </property>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>-10</y>
         <width>1200</width>
         <height>3000</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>1200</width>
         <height>3000</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>80</y>
          <width>431</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Enter your monthly income</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>160</y>
          <width>211</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string>If you register other incomes 
add their total amount:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="VenituriStraine">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>250</y>
          <width>161</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="NextVenituriWindow">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>510</y>
          <width>93</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>200</y>
          <width>254</width>
          <height>30</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Week #1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Saptamana1Salariala">
           <property name="maxLength">
            <number>12</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>280</y>
          <width>254</width>
          <height>30</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Week #2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Saptamana2Salariala"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>370</y>
          <width>254</width>
          <height>30</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Week #3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Saptamana3Salariala"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_4">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>460</y>
          <width>254</width>
          <height>30</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Week #4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Saptamana4Salariala"/>
         </item>
        </layout>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>320</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Ianuarie</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Februarie</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Martie</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="clock">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>20</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Clock</string>
        </property>
       </widget>
       <widget class="QScrollBar" name="verticalScrollBar">
        <property name="geometry">
         <rect>
          <x>790</x>
          <y>0</y>
          <width>17</width>
          <height>568</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
